src/app/staticanalysis/modules/constantFunctions.js

```diff
@@ -0,0 +1,88 @@
+var name = 'Constant functions'
+var desc = 'Check for potentially constant functions'
+var categories = require('./categories')
+var common = require('./staticAnalysisCommon')
+var callGraph = require('./functionCallGraph')
+var AbstractAst = require('./abstractAstView')
+
+function constantFunctions () {
+  this.contracts = []
+
+  constantFunctions.prototype.visit = new AbstractAst().builder(
```

Comment:
ehm i had a weird issue after the source was bablified, i could not pass this.contracts because this was not defined. It worked without babel... this was more or less a quick and dirty fix.